app/views/posts/show.html.erb

```diff
@@ -1 +1,19 @@
-<h2>details of posts<h1>
\ No newline at end of file
+<h2>details of posts<h1>
+<section class="mt-4">
+  <div class="border border-dark p-2 mb-2">
+    <div class="d-flex justify-content-between">
+      <h3>post #<%= @post.id %> by <%= @post.user.name %></h3>
+      <div class="">
+        <span>Comments: <%= @post.comments_counter || 0 %> </span>
+        <span>, Likes: <%= @post.likes_counter || 0 %></span>
+      </div>
+    </div>
+    <p><%= @post.text%></p>
+  </div>
+  <ul class="comments">
+    Comments:
+    <% @comments.each do |comment|%>
+      <li><%= comment.user.name %> : <%= comment.text %></li>
+    <% end %>
+  </ul>
+</section>
```

Comment:
- [ ] [REQUIRED] Let's also ensure that the Post show page displays the post with all comments listed as shown on the wireframe.
<img width="502" alt="Screenshot 2023-06-24 at 20 21 41" src="https://github.com/AlaaAlsalem/blogApp/assets/58692590/8f498f59-d78d-467f-98e3-c998663a3f3e">